modules/govuk/manifests/node/s_apt.pp

```diff
@@ -70,8 +70,8 @@
       release  => 'stable',
       key      => '46095ACC8548582C1A2699A9D27D666CD88E42B4';
     'grafana':
-      location => 'https://packagecloud.io/grafana/stable/ubuntu',
-      release  => 'trusty',
+      location => 'https://packagecloud.io/grafana/stable/debian',
+      release  => 'wheezy',
```

Comment:
OK, agree - looks like packagecloud have that release https://packagecloud.io/app/grafana/stable/search?dist=debian%2Fjessie